specifications/credential.issuance.protocol.md

```diff
@@ -289,19 +287,23 @@ The following is a non-normative example of a `CredentialStatus` response object
 
 ## Key Rotation and Revocation
 
-[=Issuer Service=] implementations SHOULD support rotation and revocation of keys used to
-create [=Verifiable Credential=] proofs. Key rotation and revocation may be supported in the following way:
-
-1. After a defined `cryptoperiod`, a rotation is initiated, a new key pair is generated and the public key is added
-   to a `verificationMethod` in the [=Credential Issuer=] DID document. The new private key is used to sign newly
-   issued [=Verifiable Credential=] proofs.
-2. The old private key is decommissioned (archived or destroyed). However, `verificationMethod` in
-   the [=Credential Issuer=] DID document are retained so existing issued [=Verifiable Credentials=] may be verified.
-3. At some point before existing [=Verifiable Credentials=] are set to expire, an [=Credential Issuer=] may make
-   credential offers for new [=Verifiable Credentials=] to [=Holders=].
-4. After a defined period, revocation will be performed where the public key's `verificationMethod` will be removed
-   from the [=Credential Issuer=] DID document. At this point, any existing [=Verifiable Credentials=] with proofs
-   signed by the revoked key will not verify.
+[=Issuer Service=] implementations SHOULD support rotation and revocation of keys used to create [=Verifiable Credential=] proofs. To ensure continuity and security, key rotation and revocation MAY be supported overlapping primary and secondary keys during rotation in the following way:
```

Comment:
I think this section would justify a separate discussion & PR as it expands the scope of the protocol.